includes/datatypes.php

```diff
@@ -848,3 +848,73 @@ function get_podcasting_language_options() {
 		)
 	);
 }
+
+/**
+ * Validate Podcast Taxonomy Fields.
+ *
+ * @param string $term     Term.
+ * @param string $taxonomy Taxonomy Name.
+ * @param array  $args     List of arguments.
+ *
+ * @return string
+ */
+function validate_taxonomy_fields( $term, $taxonomy, $args = [] ) {
+	// Bailout, if not the podcasts taxonomy.
+	if ( 'podcasting_podcasts' !== $taxonomy ) {
+		return $term;
+	}
+
+	$referer              = sanitize_text_field( $_POST['_wp_http_referer'] );
+	$is_onboarding_step_1 = false !== strpos( $referer, 'page=simple-podcasting-onboarding&step=1' );
```

Comment:
It may be better to use `parse_str()` here instead, the query string parameter order isn't guaranteed and another plugin may mess with it.

Reply:
correct, it is implemented https://github.com/10up/simple-podcasting/pull/216/commits/74db92ee594dd250ee5ac0c87c67c748b2d7e52f